src/_data/defaultSchedule.yml

```diff
@@ -121,21 +117,21 @@ friday:
     name: Project documentation
     url: /course/handbook/project-documentation
   - start: 14:30
-    end: 15:15
+    end: 15:00
+    name: Presentation prep
+    url: /course/handbook/projects/#project-presentation
+  - start: 15:00
+    end: 15:30
+    name: Team SGC
+    url: /course/handbook/retrospectives/#team-retrospectives
+  - start: 15:30
+    end: 16:15
     name: Presentations
     type: presentation
-  - start: 15:15
-    end: 16:00
-    name: Cohort SGC
-    url: /course/handbook/retrospectives/
-  - start: 16:00
-    end: 16:45
-    name: Team SGCs
-    url: /course/handbook/retrospectives/#team-retrospectives
-  - start: 16:45
+  - start: 16:15
```

Comment:
Is 45 minutes enough for presentations?

In [our guidance](https://learn.foundersandcoders.com/course/handbook/project-presentations/), we recommend 12+5 minutes per presentation, so 51 minutes for three groups.

Reply:
has been enough so far!! maybe we should change our guidance to 12+3